server/routes/serviceProviderReferrals/endOfServiceReportCheckAnswersPresenter.ts

```diff
@@ -32,6 +20,7 @@ export default class EndOfServiceReportCheckAnswersPresenter {
   readonly answersPresenter = new EndOfServiceReportAnswersPresenter(
     this.referral,
     this.endOfServiceReport,
-    this.serviceCategory
+    this.serviceCategory,
+    true
```

Comment:
Does this need to be here if we've made `true` the default argument in the presenter? Or do you think it's better for readability here?

Reply:
Yeah, I added the default argument only to be able to skip it in the tests where it made no difference; I think it's good to be explicit about the value.